feat: ADR-013: Per-chain default external address (#887)
desmos-labs/desmos: x/profiles/spec/07-client.md

  total: "0"
```

#### default-external-addresses

> The `default-external-addresses` command allows users to query for default external addresses in chain link optionally specifying an owner address and a chain name.

Reviewer: ```suggestion
The `default-external-addresses` command allows users to query for default external addresses optionally specifying an owner address and a chain name.
```

desmos tx profiles unlink-chain "cosmos" cosmos18xnmlzqrqr6zt526pnczxe65zk3f4xgmndpxn2`
```

#### set-default-external-address

> The `set-default-external-address` allows user to set a default external address.

Reviewer: ```suggestion
The `set-default-external-address` command allows user to set a default external address.
```

}
```

### DefaultExternalAddresses

> The `DefaultExternalAddresses` endpoint allows user to query for external default address optionally specifying an owner and chain name.

Reviewer: ```suggestion
The `DefaultExternalAddresses` endpoint allows to query for external default addresses optionally specifying an owner and chain name.
```

```

**Notes**
- The `chainName` parameter will only be used if the `owner` one is specified

Example
```bash
grpcurl -plaintext  \
  -d '{"owner": "desmos1evj20rymftvecmgn8t0xv700wkjlgucwfy4f0c", "chainName": "cosmos"}' localhost:9090 desmos.profiles.v3.Query/DefaultExternalAddresses
```

Example Output:
```json
{
  "links": [
    {
      "user": "desmos1evj20rymftvecmgn8t0xv700wkjlgucwfy4f0c",
      "address": {"@type":"/desmos.profiles.v3.Bech32Address","prefix":"cosmos","value":"cosmos13n9wek2ktpxhpgfrd39zlaqaeahxuyusxrsfvn"},
      "proof": {
        "pubKey": {"@type":"/cosmos.crypto.secp256k1.PubKey","key":"AqYZhHKaeBcrYktZEvor/SUDlHCkv5JBplaG2vc2bvfS"},
        "signature": {"@type":"/desmos.profiles.v3.SingleSignatureData","mode":"SIGN_MODE_DIRECT","signature":"3yCU8/HKv7Vn0sf7HB+AhV/hK37DCBAkQkXdruaFSvMjRZB1XrYkpKWZVi+xnhSenc1p951Q1058rrYjuNCk9g=="},
        "plainText": "6465736d6f733165766a323072796d66747665636d676e3874307876373030776b6a6c67756377667934663063"
      },
      "chainConfig": {
        "name": "cosmos"
      },
      "creationTime": "2022-07-18T10:07:51.899288600Z"
    }
  ],
  "pagination": {
    "total": "1"
  }
}
```

> 

Reviewer 2: Can you also add the REST endpoint inside the REST section?
